math256/math256_cfg.json

```diff
@@ -0,0 +1,4 @@
+{
+    "name": "math256",
+    "mem_page_count": 1

```

Comment:
you probably want to increase this. right now, fldist defaults to mem_page_count=100 but that should change. ditto for Config.toml

Reply:
i am not sure here. i though in simple programs (services) like this do not require memory so much? so the less is better